checker/manual/constant-value-checker.tex

```diff
@@ -185,22 +217,31 @@ \section{Warnings\label{value-checker-warnings}}
 \end{itemize}
 \end{sloppypar}
 
-There is one other situation in which the Constant Value Checker produces a
+There are some other situations in which the Constant Value Checker produces a
 warning message:
 
 \begin{sloppypar}
 \begin{itemize}
-\item \code{[too.many.values.given] Annotation ignored because the maximum
-  number of values tracked is 10.}
+\item \code{[too.many.values.given] The maximum number of arguments permitted is 10. Remove the annotation. Treating result as @UnknownVal}
 
   The Constant Value Checker only tracks up to 10 possible values for an
   expression.  If you write an annotation with more values than will be
-  tracked, the annotation is ignored.
+  tracked, the annotation is replaced with \<@UnknownVal>.
 
-\end{itemize}
-\end{sloppypar}
+\item \code{[too.many.int.values] The maximum number of integer arguments permitted is 10. Remove the annotation. Treating result as @IntRange to reduce computational complexity}
```

Comment:
"to reduce computational complexity" seems irrelevant here.  I don't think it's a good use of space.